demisto_sdk/commands/common/docker_helper.py

```diff
@@ -159,12 +182,37 @@ def create_container(
             self.copy_files_container(container, files_to_push)
         return container
 
+    def push_image(self, image: str, log_prompt: str = ""):
+        for _ in range(2):
+            try:
+
+                test_image_name_to_push = image.replace(
+                    "docker-io.art.code.pan.run/", ""
+                )
+                docker_push_output = init_global_docker_client().images.push(
+                    test_image_name_to_push
+                )
+                logger.info(
+                    f"{log_prompt} - Trying to push Image {test_image_name_to_push} to repository. Output = {docker_push_output}"
+                )
+                break
+            except (
+                requests.exceptions.ConnectionError,
+                urllib3.exceptions.ReadTimeoutError,
+                requests.exceptions.ReadTimeout,
+            ):
+                logger.info(
```

Comment:
warning/error? or is it expected at some flows?

Reply:
changed to warning